pkg/crc/machine/start.go

```diff
@@ -713,3 +706,24 @@ func ensureRoutesControllerIsRunning(sshRunner *crcssh.Runner, ocConfig oc.Confi
 	}
 	return nil
 }
+
+func updateKubeconfig(ocConfig oc.Config, sshRunner *crcssh.Runner, srcKubeconfigFilePath, destKubeconfigPath string) error {
+	if _, err := os.Stat(destKubeconfigPath); err == nil {
+		return nil
+	}
+	selfSignedCAKey, selfSignedCACert, err := crctls.GetSelfSignedCA()
+	if err != nil {
+		return errors.Wrap(err, "Not able to generate root CA key and Cert")
+	}
+	if err := copyKubeconfigFileWithUpdatedUserClientCertAndKey(selfSignedCAKey, selfSignedCACert, srcKubeconfigFilePath, destKubeconfigPath); err != nil {
+		return errors.Wrapf(err, "Failed to copy kubeconfig file: %s", constants.KubeconfigFilePath)
+	}
+	adminClientCA, err := adminClientCertificate(constants.KubeconfigFilePath)
```

Comment:
This is not a CA, this is a certificate.

Reply:
@cfergeau resolved, missed this one :(